src/treelearner/split_info.hpp

```diff
@@ -49,7 +49,7 @@ struct SplitInfo {
   bool default_left = true;
   int8_t monotone_type = 0;
   inline static int Size(int max_cat_threshold) {
-    return 2 * sizeof(int) + sizeof(uint32_t) + sizeof(bool) + sizeof(double) * 9 + sizeof(data_size_t) * 2 + max_cat_threshold * sizeof(uint32_t) + sizeof(int8_t);
+    return 2 * sizeof(int) + sizeof(uint32_t) + sizeof(bool) + sizeof(double) * 7 + sizeof(data_size_t) * 2 + max_cat_threshold * sizeof(uint32_t) + sizeof(int8_t);
```

Comment:
minor bugfix -- I noticed min_constraint/max_constraint were removed, but the size calculation was not updated:

https://github.com/microsoft/LightGBM/pull/1314/files

note this doesn't fix any bug, I just noticed this by chance and updated

Reply:
it looks like this is the PR that removed them but didn't update the size calculation:
https://github.com/microsoft/LightGBM/commit/3670e476b1918e72e72ccf00bca69d398827e24f#diff-85924b6a07e712436ab5e7782288e2a7